symtabAPI/h/Symtab.h

```diff
@@ -111,6 +91,9 @@ class SYMTAB_EXPORT Symtab : public LookupInterface,
    friend class relocationEntry;
    friend class Object;
 
+   // Hide implementation details that are complex or add large dependencies
+   std::unique_ptr<symtab_impl> impl;
```

Comment:
`const std::unique_ptr...`?  The impl never changes.

Reply:
Since `Symtab` isn't moveable, that is a good idea.